docs/pages/distribution/app-stores.md

```diff
@@ -52,11 +52,29 @@ To access these values at runtime, you can use the [Expo Constants API](../versi
 
 ## iOS-specific guidelines
 
-- All apps in the iTunes Store must abide by the [App Store Review Guidelines](https://developer.apple.com/app-store/review/guidelines/).
-- Apple will ask you whether your app uses the IDFA, the answer is "yes." This is because Expo contains the Facebook and Branch SDKs, which contain code for collecting the IDFA, and you'll need to check a couple boxes on the Apple submission form. See [Branch's Guide](https://blog.branch.io/how-to-submit-an-app-that-uses-an-idfa-to-the-app-store/) for which specific boxes to fill in.
+- All apps in the App Store must abide by the [App Store Review Guidelines](https://developer.apple.com/app-store/review/guidelines/).
+- Apple will ask you whether your app uses the IDFA: the answer is "yes." This is because Expo contains the Facebook and Branch SDKs, which contain code for collecting the IDFA, and you'll need to check a couple boxes on the Apple submission form. See [Branch's Guide](https://blog.branch.io/how-to-submit-an-app-that-uses-an-idfa-to-the-app-store/) for which specific boxes to fill in.
 
 > **Note**: No data is sent to Branch, Facebook, Segment, or Amplitude from your app unless you explicitly do so using the APIs. For more information on how Expo handles your data, and your end users' data, take a look at our [Privacy Explained page](https://expo.io/privacy-explained).
 
+### App privacy questions
+
+Beginning December 8, 2020, new app submissions and updates are required to provide information about their privacy practices in App Store Connect. See [App privacy details on the App Store](https://developer.apple.com/app-store/app-privacy-details/) for more information.
+
+Apple will ask you a series of questions:
+
+- Select **Yes, we collect data from this app**. Click **Next**.
+- Select **Device ID**
+    - Managed standalone apps include the Facebook, Facebook Ads, and Google AdMob SDKs, which still access the IDFA.
```

Comment:
What do you mean by "access the IDFA"? Is the IDFA ever used if I do not explicitly use Facebook or Google SDKs? And in this case, is the IDFA ever transmitted over the network?

If yes that's a quite huge privacy concern.

Reply:
My understanding is that these libraries transmit i.e. collect the IDFA only when used. If you don't use these APIs then the IDFA is never collected. You can use e.g. Charles Proxy to confirm this.

It may still be prudent to mark that your app accesses a device ID since the libraries are still included, erring on the side of transparency. App review is inconsistent, though, so there isn't a clear-cut answer.

Reply:
Hi @ide. Thank you for your answer, I will sniff the network as you suggested and see for myself.

Maybe a suggestion: it might be good for developers to know exactly what is collected by which SDK, it's currently quite difficult to follow all the changes related to the IDFA (reading the changelogs I understood that Amplitude has been updated to not collect the IDFA for managed apps, what about Segment, and other SDKs?). A documentation page that lists the SDKs and the data they need would be appreciated.